changelog-endpoint/README.md

```diff
@@ -0,0 +1,36 @@
+# Changelog endpoint
+The `changelog-endpoint` module is a REST service intended for Iroha data model migrations: adding signatories, setting account details, creating domains, appending roles etc.
+
+## Simplified flow
+1) The service executes given Groovy script that implements `ChangelogInterface`.
+2) Created changelog transaction(generated by Groovy script) is accompanied with a changelog history transaction in order to keep track of changes.
+3) Then, transactions are combined into an atomic batch. 
+4) Superuser signs recently created batch and sends it to Iroha as MST. 
+
+## Brief API documentation
+Available endpoints:
+1) `/changelog/changelogFile` - executes changelog Groovy script file which absolute path is defined in a JSON request.  
```

Comment:
executes changelog Groovy script file absolute path of which is defined in a JSON request.